apps/fluid-dev/src/pages/checkbox/checkbox-page.component.html

```diff
@@ -3,6 +3,7 @@
   [disabled]="disabled"
   [checked]="_isChecked()"
   [indeterminate]="_isIndeterminate()"
+  [value]="'checkbox'"
```

Comment:
No need to bind that property, if you don't bind it to a variable.
```suggestion
  value="checkbox"
```